locale/pl/wivrn-dashboard.po

```diff
@@ -0,0 +1,971 @@
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the WiVRn-dashboard package.
+#
+# SPDX-FileCopyrightText: 2025 polyborea <patryklimerick@gmail.com>
+msgid ""
+msgstr ""
+"Project-Id-Version: WiVRn-dashboard\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2025-11-11 21:12+0100\n"
+"PO-Revision-Date: 2025-11-13 23:58+0100\n"
+"Last-Translator: polyborea <patryklimerick@gmail.com>\n"
+"Language-Team: Polish <kde-i18n-doc@kde.org>\n"
```

Comment:
Lokalize insists in setting it to KDE's contacts, just remove set it to "Language-Team: Polish <>", same for client file.